evaluation/run_evaluation_harness.py

```
@@ -76,6 +85,33 @@ def _write_to_model_csv(self, path: Path, model_name: str, batch: list[dict[str,
+    def _process_batch(
+        self, 
+        texts: list[str], 
+        path: Path, 
+        model: BaseModel, 
+        model_name: str, 
+        batch: list[dict[str, str | int]],
+    ) -> None:
+        predictions = model.batch_classify(texts)
+        self._write_to_model_csv(path, model_name, batch, predictions)
+
+    def _write_to_deadletter_csv(self, path, model_name, batch):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_write_to_deadletter_csv`

Add return type annotation: `None`

(ANN202)